lib/commands/context.js

```diff
@@ -61,6 +61,16 @@ commands.setContext = async function setContext (name) {
   this.curContext = name;
 };
 
+commands.mobileGetContexts = async function mobileGetContexts () {
```

Comment:
Please add a dosctring here that describes the returned result and its format (preferable with a JSON example)

Reply:
Thanks. I have added a doc string there. Could you check it?